app/controllers/application_controller.rb

```diff
@@ -54,4 +54,14 @@ def require_authenticated_user
       redirect_to main_app.user_omniauth_authorize_path(:shibboleth)
     end
   end
+
+  def authorize_report_access
```

Comment:
I think this is just unfortunately how this gem works, @purelogiq can probably clarify.
https://github.com/tma1/dossier/commit/1aa3ea105b91046c2d04ef0af9836445c6d6a92c

This gem does exactly what we need pretty easily though, so I think this is justified.